dotnet/Directory.Packages.props

```diff
@@ -39,7 +39,12 @@
     <PackageVersion Include="Microsoft.AspNetCore.App" Version="8.0.4" />
     <PackageVersion Include="Microsoft.AspNetCore.OpenApi" Version="8.0.8" />
     <PackageVersion Include="Microsoft.ApplicationInsights.AspNetCore" Version="2.22.0" />
-    <PackageVersion Include="Microsoft.Bcl.AsyncInterfaces" Version="8.0.0" />
+    <PackageVersion Include="Microsoft.Bcl.AsyncInterfaces" Version="9.0.0-rc.2.24473.5" />
```

Comment:
Suggest: Pull out the MEAI version into a MSBuild variable so it can be shared. Similar to how `MicrosoftSemanticKernelExperimentalVersion` does.

Reply:
cool, will do!